packages/app/features/top/ChannelScreen.tsx

```diff
@@ -71,14 +71,22 @@ export default function ChannelScreen(props: Props) {
         store.syncChannelThreadUnreads(channel.id, {
           priority: store.SyncPriority.High,
         });
+        if (group) {
+          // Update the last visited channel in the group so we can return to it
+          // when we come back to the group
+          db.updateLastVisitedChannelInGroup({
```

Comment:
Could this call be part of `markChannelVisited`?

Reply:
The issue with using markChannelVisited is that we only call it when the user navigates away from it, I think we should update last visited as soon as a user navigates into a channel to avoid any weirdness that might crop up if the cleanup function in useFocusEffect doesn't run for some reason.

Reply:
Ok skip it -- actually I think the channel visitedAt is redundant in my branch anyway (it's only used for the welcome message, and I'm not going to hide the welcome message until there are actually posts), so I'll probably remove it.